### PR TITLE
Can now set a FEN position.

### PR DESCRIPTION
**Pull request by hitecherik**
I needed functionality to set a FEN position in one of my own projects; so I forked this and added a set_fen_position function function to the Stockfish class.

### CODE REVIEW

File: stockfish.py [hunk 1]
```diff
@@ -98,6 +98,9 @@ def set_position(self, moves=None):
             moves = []
         self.__put('position startpos moves %s' %
                    self.__convert_move_list_to_str(moves))
+    
+    def set_fen_position(self, fen):
```
Comment (zhelyabuzhsky) [inline stockfish.py]: please, rename parameter "fen" to "fen_position"

Comment (zhelyabuzhsky) [reply stockfish.py]: please, add this method to README.md and write test on this method

Comment (hitecherik) [reply stockfish.py]: Ok - will do soon.

**Comment by hitecherik** [comment]
Made requested changes.

**Comment by zhelyabuzhsky** [comment]
Well done

**Pull request merged**